lib/TPP/Dialect/Tpp/TppUtils.cpp

```diff
@@ -363,8 +403,7 @@ bool isTppAdd(linalg::GenericOp linalgOp) {
     return false;
   if (!allOperandsHaveSameShapeAndStrides(linalgOp->getOperands().getTypes()))
     return false;
-  return allIndexingsAreProjectedPermutation(linalgOp) &&
-         hasOnlyOp<arith::AddFOp>(linalgOp.getRegion());
+  return hasOnlyOp<arith::AddFOp>(linalgOp.getRegion());
```

Comment:
Why this change?

Reply:
We use projectedPermutation for each tpp. So now we check directly in `hasMappingToTppConditions`